civitai-downloader-v2/src/core/category/category_classifier.py

```
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""
+Category Classifier - 複数カテゴリを持つモデルの分類機能
+優先順位に基づいて主カテゴリを決定する
+"""
+
+from typing import List, Dict, Optional, Tuple, Any
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)